hydra/test_utils/test_utils.py

```diff
@@ -306,7 +328,10 @@ def experiment(cfg):
             modified_env.update(env_override)
         subprocess.check_call(cmd, env=modified_env)
 
-        with open(output_file, "r") as f:
+        if post_run_callback:
+            post_run_callback(**locals())
```

Comment:
make the callback well defined and accept specific things. passing **locals()  is fragile and unclear.

Reply:
true, i will make this one better.